app/src/main/java/com/example/weather/ui/Forecast/ForecastItemAdapter.java

```diff
@@ -1,31 +1,39 @@
 package com.example.weather.ui.Forecast;
 
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.util.Log;
 import android.view.LayoutInflater;
+import android.view.View;
 import android.view.ViewGroup;
 
 import androidx.annotation.NonNull;
 import androidx.recyclerview.widget.RecyclerView;
 
+import com.example.weather.data.db.entity.WeatherEntity;
 import com.example.weather.databinding.ItemForecastBinding;
+import com.example.weather.ui.Temp.TempActivity;
 
 import java.util.List;
 
 public class ForecastItemAdapter extends RecyclerView.Adapter<ForecastItemAdapter.MyViewHolder> {
-    private final List<String> forecasts;
+    private final List<WeatherEntity> forecasts;
 
-    public ForecastItemAdapter(List<String> forecast){
+    public ForecastItemAdapter(List<WeatherEntity> forecast) {
         this.forecasts = forecast;
     }
 
     @NonNull
     @Override
     public MyViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
-        return new MyViewHolder(ItemForecastBinding.inflate(LayoutInflater.from(parent.getContext()),parent,false));
+        return new MyViewHolder(ItemForecastBinding.inflate(LayoutInflater.from(parent.getContext()), parent, false));
     }
 
     @Override
     public void onBindViewHolder(@NonNull MyViewHolder holder, int position) {
-        holder.bind("123",position);
+        holder.bind(forecasts.get(position).getName(), position);
```

Comment:
Why we can't pass full `WeatherEntity` object to bind(). In the future you'll use not only name, temperature e.g.